concordium-std-derive/src/lib.rs

```diff
@@ -1048,6 +1060,37 @@ fn contract_function_optional_args_tokens(
     (setup_fn_args, fn_args)
 }
 
+#[cfg(feature = "build-schema")]
+fn contract_function_schema_tokens2(
```

Comment:
Perhaps name this with something with `event` in the name. the `2` suffix does not say very much.